File: proyectomds2/frontend/src/vista-buscador-listas-reproduccion.js
import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
import '@vaadin/vaadin-ordered-layout/src/vaadin-vertical-layout.js';

class VistaBuscadorListasReproduccion extends PolymerElement {

    static get template() {
        return html`
<style include="shared-styles">
                :host {
                    display: block;
                    height: 100%;
                }
            </style>
<vaadin-vertical-layout style="width: 100%; height: 100%;" id="vaadinVerticalLayout">
 <label style="width: 100%;" id="titulo_listas_reproduccion">Listas de reproducción con la canción </label>
 <vaadin-vertical-layout theme="spacing" style="width: 100%; height: 100%;" id="lista_reproduccion_buscada"></vaadin-vertical-layout>
</vaadin-vertical-layout>
`;
    }

    static get is() {
        return 'vista-buscador-listas-reproduccion';
    }

    static get properties() {
        return {
            // Declare your properties here.
        };
    }
}

customElements.define(VistaBuscadorListasReproduccion.is, VistaBuscadorListasReproduccion);

File: proyectomds2/src/main/java/vistas/VistaBuscadorListasReproduccion.java
package vistas;

import com.vaadin.flow.templatemodel.TemplateModel;
import com.vaadin.flow.component.Tag;
import com.vaadin.flow.component.dependency.JsModule;
import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
import com.vaadin.flow.dom.Element;
import com.vaadin.flow.component.template.Id;
import com.vaadin.flow.component.html.Label;

/**
 * A Designer generated component for the vista-buscador-listas-reproduccion template.
 *
 * Designer will add and remove fields with @Id mappings but
 * does not overwrite or otherwise change this file.
 */
@Tag("vista-buscador-listas-reproduccion")
@JsModule("./src/vista-buscador-listas-reproduccion.js")
public class VistaBuscadorListasReproduccion extends PolymerTemplate<VistaBuscadorListasReproduccion.VistaBuscadorListasReproduccionModel> {

    @Id("vaadinVerticalLayout")
	private Element vaadinVerticalLayout;
	@Id("titulo_listas_reproduccion")
	private Label titulo_listas_reproduccion;
	@Id("lista_reproduccion_buscada")
	private Element lista_reproduccion_buscada;

	/**
     * Creates a new VistaBuscadorListasReproduccion.
     */
    public VistaBuscadorListasReproduccion() {
        // You can initialise any data required for the connected UI components here.
    }

    /**
     * This model binds properties between VistaBuscadorListasReproduccion and vista-buscador-listas-reproduccion
     */
    public interface VistaBuscadorListasReproduccionModel extends TemplateModel {
        // Add setters and getters for template properties here.
    }
}
